flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -63,9 +65,14 @@
 	 */
 	public static final ConfigOption<Integer> VCORES =
 		key("yarn.containers.vcores")
-		.defaultValue(-1)
-		.withDescription("The number of virtual cores (vcores) per YARN container. By default, the number of vcores" +
-			" is set to the number of slots per TaskManager, if set, or to 1, otherwise.");
+			.defaultValue(-1)
+			.withDescription(Description.builder().text(
+					"The number of virtual cores (vcores) per YARN container. By default, the number of vcores" +
+					" is set to the number of slots per TaskManager, if set, or to 1, otherwise. In order for this" +
+					" parameter to be used your cluster must have CPU scheduling enabled. You can do this by setting" +
+					" the %s.",
+				code("org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler"))
```

Comment:
I guess this works now but does it make sense to continue with the `DescriptionBuilder`? After all we could add the `markdown="span"` to the `td` tag, or configure `parse_block_html`, and simply accept markdown syntax in a string.

Reply:
@GJL thanks for your comments.The original reason for introducing the DescriptionBuilder was not to embed html or markdown into the description so that we can format it differently in case we want to print it in Exception description.